.circleci/config.yml

```diff
@@ -9,11 +10,6 @@ e2eTest: &e2eTest
 
     - run: ./e2e-harness setup --name=ci-awsop-${CIRCLE_SHA1:0:7}
 
-    - run:
-        name: additional setup required for compiling go on circleCI
-        command: |
-          mkdir -p /home/circleci/.go_workspace/src
-          cp -ar vendor/* /home/circleci/.go_workspace/src/
```

Comment:
Why `working_directory` is needed and this is removed?

Reply:
It is simpler to put the working directory inside `GOPATH` than copying the vendor directory there from another location, single directive and no copying involved.